app.py

```diff
@@ -249,27 +279,60 @@ def _get_featured_post(groups=[], categories=[], per_page=1):
 
     return posts[0] if posts else None
 
+def _get_category_by_id(category_id):
+    global CATEGORIESBYID
+    category = CATEGORIESBYID[category_id]['name']
+    return category
+
+def _get_group_by_id(group_id):
+    global GROUPBYID
+    group = {
+        'name': GROUPBYID[group_id]['name'],
+        'slug': GROUPBYID[group_id]['slug'],
+    }
+    return group
```

Comment:
Couldn't you just return `GROUPBYID[group_id]` as the object is structured the same way as the one you are building here.